### DISCUSSION TITLE
table UI

### DISCUSSION THREAD

**Discussion by koten551** [heroui-inc/heroui#550] [Feedback]
using input tag on Table.cell making this is read only, cant change value of <input>

**Comment by jrgarciadev** [comment]
Hey @koten551, NextUI Table component is based on https://react-spectrum.adobe.com/react-spectrum/TableView.html which is not meant to be edited in the same row, I suggest you have an action that opens up a modal where the user can edit the row data

Hope it helps you  🙌🏻
